tests/atlocal.in

```diff
@@ -175,6 +175,10 @@ if test "$GNUCOBOL_ENV_SETUP" != "1" -a "$GNUCOBOL_TEST_LOCAL" != "1"; then
 	SHLIB_PATH="@abs_top_builddir@/libcob/.libs:@JNI_LIBPATH@${SHLIB_PATH}"
 	LIBPATH="@abs_top_builddir@/libcob/.libs:@JNI_LIBPATH@${LIBPATH}"
 	COB_LIBRARY_PATH="@abs_top_builddir@/extras"
+	
+	if test "x$MSYSTEM" != "x" -o "$OSTYPE" = "cygwin"; then
+	   COB_LIBRARY_PATH="${COB_LIBRARY_PATH}:@abs_top_builddir@/libcob/.libs"
+	fi
```

Comment:
the libcob delay-load libraries should not use COB_LIBRARY_PATH but the system path (commonly `%PATHZ%`/ `$LD_LIBRARY_PATH`), if this isn't the case then `cob_load_lib()` needs to be adjusted, if this is the case and it doesn't work, then setting those variables in this file (or for msys: translating them correctly) needs to be fixed

Reply:
Yes that's what I was thinking. However those paths seemed to be set correctly at a first glance; however I'm still investigating.

Reply:
Note: a not working delay-load can also be related to the dependencies not being in PATH - but lterror (=merging the referenced commit from 3.x to trunk and applying it to `cob_load_lib()` will tell you what the real issue is (which may be useful before analyzing the cause).

Reply:
Ok apparently the directory where `jvm.dll` is located was not in `PATH`. I've fixed that in the CI workflow, but that may need to be done in `atlocal` instead.